src/modules/invoices/services/refund-requests.service.ts

```
@@ -0,0 +1,297 @@
+import { getLogger } from '@logtape/logtape';
+import { refundRequestsQueries } from '@/modules/invoices/database/queries/refund-requests.queries';
+import type { SelectRefundRequest } from '@/modules/invoices/database/schema/refund-requests.schema';
+import { invoicesRepository } from '@/modules/invoices/database/queries/invoices.repository';
```

ESLint

[error] `@/modules/invoices/database/queries/invoices.repository` import should occur before import of `@/modules/invoices/database/queries/refund-requests.queries`

(import/order)